src/database/ClassDatabaseController.java

```diff
@@ -0,0 +1,77 @@
+package database;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import cellItems.ClassCellItems;
+
+public class ClassDatabaseController {
+	public static List<ClassCellItems> getAllClasses(){
+		Connection conn = null;
+		try{
+			conn = MyDBConnection.getConnection();
+			Statement st = conn.createStatement();
+			ResultSet rs = st.executeQuery("Select * from Subject");
+			List<ClassCellItems> classes = new ArrayList<>();
+			while(rs.next()){
+				PreparedStatement ps = conn.prepareStatement("Select Subject_time.* from Subject_time Join Subject_times on Subject_time.IdSubject_time = Subject_times.IdSubject_time where Subject_times.IdSubject = ?");
```

Comment:
These ones will be causing trouble. It should be `Subject_Time` and `Subject_Times` instead of `Subject_time` and `Subject_times`